build/generate-crds.sh

```diff
@@ -32,7 +32,7 @@ function generateCRDs() {
   # Add maistra-version label
   sed -i -e "s/^  annotations:/  labels:\n    maistra-version: $MAISTRA_VERSION\n\\0/" deploy/crds/*
 
-  sed -i -e 's/^  annotations:/  annotations:\n    service.beta.openshift.io\/inject-cabundle: "true"/' \
+  sed -i -e 's/^  annotations:/  annotations:\n    service.beta.openshift.io\/inject-cabundle: "false"/' \
       deploy/crds/maistra.io_servicemeshcontrolplanes.yaml
```

Comment:
Instead of removing this annotation, we set it to false to ensure that when the CRD manifest is applied to a cluster that already has this CRD, the annotation is actually overwritten. I believe that if we simply removed this annotation from the manifest, the annotation wouldn't be removed.